websites/S/Salad/presence.ts

```diff
@@ -0,0 +1,100 @@
+const presence = new Presence({
+		clientId: "911546577485725706",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+	
+
+const enum Assets {
+	Logo = "https://cdn.solo.to/user/a/63654eae75b1e7_99721437.gif",
+}
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: Assets.Logo,
+		startTimestamp: browsingTimestamp,
+	};
+	const privacy = await presence.getSetting<boolean>("privacy");
+
+	if (document.location.pathname.includes("/earn/summary")) {
```

Comment:
Simplify it, do this for all mentions of this.
```suggestion
	if (pathname.includes("/earn/summary")) {
```